_posts/2016-01-30-css-essentials.md

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: CSS-essentials, Onefootball open source CSS project
 permalink: /onefootball-open-source-css-essentails
-tags: css, flex, onefootball, css-essentials, open source
+tags: [css, flex, onefootball, css-essentials, open-source]
```

Comment:
Since the tags are made of couples key-vaule with key a single word and value whatever we want (value is the actual string displayed in the website). 

We have no limit here :) for simplicity I'd go with all lowercase.

Then in the descriptions we can use whatever we think is more appropriated.